### ISSUE TITLE
manually add entry to task

### ISSUE THREAD

**Issue by tobehn** [lakoliu/Furtherance#83]
I want to be able to manually add a timetrack entry to a task in case i forgot to track a period of time

1. Click on Task
2. Plus Button
3. Define _start_time_
4. define amount of time (_stop_time_ calculates by function)
5. click save

**Comment by lakoliu** [comment]
Hi @tobehn. You should be able to do this already if I understand correctly. At the top left corner of Furtherance there is a **+**. Click that and you can manually add a task. It's not exactly as you mentioned, because you have to type in the name of the task and the tags you want, but it should function the same. If you already have a task with the same name and tags on that day it will automatically be grouped with it. Please let me know if I misunderstood.

**Comment by tobehn** [comment]
Hi there @lakoliu,

i get it now. Though it would be awesome to do this within a task so i do not need to type in the whole task again (especially with typos or missing tags i would then create an additional entry rather than a subentry within the chosen task.

I often have to add an entry to a previously defined task. As i love the task centric approach of furtherance i think it would be a great feature.

**Comment by lakoliu** [comment]
I think that's a good idea. I will look into adding an add button to the task details page that auto-fills the task name and tags with the same information. Thanks!

**Comment by tobehn** [comment]
I just went into Workbench this morning and played around with the *.ui files
basically i'd suggest to add an *add entry* button to the task_detail.ui

![Bildschirmfoto vom 2022-10-16 10-40-06](https://user-images.githubusercontent.com/40893107/196026558-6c14b684-dc0a-4dfd-89c2-157281d7f22c.png)

this opens the MessageDialog for editing the task except start and stop values are set not set and have to be choosen to validate the new entry while pressing OK. 

![Bildschirmfoto vom 2022-10-16 10-57-51](https://user-images.githubusercontent.com/40893107/196027161-7b3f5212-3e2e-488a-ab45-4039bc499580.png)

What do you think about it?

**Comment by lakoliu** [comment]
Yes, I think that looks good. I'll probably move the add button down near the delete button, but other than that I think this works. Sorry it's taken me so long to get to this, I've been working on other projects, but I'll try to get it done this week.

**Comment by lakoliu** [comment]
Just added this feature. It will be in the next release. Here is what it looks like:

![Screenshot of Add Similar button](https://user-images.githubusercontent.com/99976966/196400469-edc1ea38-5d1a-43e5-af00-214abc0fbc69.png)

**Comment by tobehn** [comment]
Hey @lakoliu,

thank you very much. I am looking forward to check it out as soon as it is released.
I appreciate you implemented it!

**Comment by tobehn** [comment]
Hi Ricky,
i know you are up to new adventures and trying to learn new stuff. 

I really would appretiate to see this feature in the app. 

Would you mind to release a version to flathub with the latest code. 
All the best to you

**Comment by lakoliu** [comment]
Yes! I'm very sorry for the delay! I will release it as soon as possible, definitely this week.
